Eshop.Domain/Eshop.Domain.csproj

```diff
@@ -6,4 +6,8 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Remove this, it was not needed for Category so why we need it for Product?

Reply:
I did not know it is not needed, I dont even know there is something like that

Reply:
but removed